src/remove.js

```diff
@@ -7,17 +7,19 @@
  * @module DeleteItem
  * @see https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_DeleteItem.html
  */
-const { apply, bind, compose, curry, pipeP } = require('ramda');
+const { apply, bind, compose, curry, pipeWith, andThen } = require('ramda');
 const { unwrapProp } = require('./wrapper');
 const addTableName = require('./table-name');
 const { mapMergeFirstPairOfArgs } = require('./map-merge-args');
 const generateKey = require('./generate-key');
 const addReturnValues = require('./return-values');
+const pipeP = pipeWith(andThen);
 
 /**
  * @private
  */
-const removeAndUnwrapAttributes = deleteItem => pipeP(apply(deleteItem), unwrapProp('Attributes'));
+const removeAndUnwrapAttributes = deleteItem =>
+  pipeP([apply(deleteItem), unwrapProp('Attributes')]);
```

Comment:
@nfantone does this makes sense to you? I can move the pipeP to a util folder before merging.

Reply:
Makes sense to me. Although, in hindsight, I would probably use a regular `async` function now.

Reply:
Something like [this](https://github.com/flybondi/flynamo/pull/51/commits/010863e6cb7bc5e9f33fb03011b05592fafa198f)?